templates/vue/src/components/TapestryNode/index.vue

```diff
@@ -121,15 +121,15 @@
 <script>
 import * as d3 from "d3"
 import { mapActions, mapGetters, mapState, mapMutations } from "vuex"
-import TapestryIcon from "@/components/TapestryIcon"
+import TapestryIcon from "@/components/common/TapestryIcon"
 import { names } from "@/config/routes"
 import { bus } from "@/utils/event-bus"
 import Helpers from "@/utils/Helpers"
 import * as wp from "@/services/wp"
-import AddChildButton from "./tapestry-node/AddChildButton"
-import ProgressBar from "./tapestry-node/ProgressBar"
-import StatusBar from "./tapestry-node/StatusBar"
-import NodeButton from "./tapestry-node/NodeButton"
+import AddChildButton from "@/components/TapestryNode/AddChildButton"
+import ProgressBar from "@/components/TapestryNode/ProgressBar"
+import StatusBar from "@/components/TapestryNode/StatusBar"
+import NodeButton from "@/components/TapestryNode/NodeButton"
```

Comment:
I see the idea behind changing these to absolute paths, but I think for these components that live in the same folder relative imports are nicer e.g. `./AddChildButton`instead.

Reply:
Agreed. Although not the biggest deal.